### PR TITLE
arpack 3.3.0

### PR DESCRIPTION
**Pull request by ilovezfs**
- always run the tests
- style changes

### CODE REVIEW

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
Seems ready to go.

**Comment by ilovezfs** [comment]
@dpo, yes although I suspect @sjackman may want the style of https://github.com/Homebrew/homebrew-science/pull/3781/files#diff-694b5d3d75fbe231fa066e8800056934R63, and following, tweaked.

**Comment by sjackman** [comment]
It's fine by me.

**Comment by ilovezfs** [comment]
Oh you're just saying that! Don't be shy!

File: arpack.rb [hunk 1]
```diff
@@ -23,37 +21,56 @@ class Arpack < Formula
...
+      %w[
+        pcndrv1
+        pdndrv1
+        pdndrv3
+        pdsdrv1
+        psndrv1
+        psndrv3
+        pssdrv1
+        pzndrv1
+      ].each { |slv| system "mpirun", "-np", "4", slv }
```
Comment (sjackman) [inline arpack.rb]: I'd be inclined to put the brackets on a single line. Other than that, it looks good to me.
```ruby
%w[pcndrv1 pdndrv1 pdndrv3 pdsdrv1 psndrv1 psndrv3 pssdrv1 pzndrv1].each do |slv|
  system "mpirun", "-np", "4", slv
end
```

Comment (ilovezfs) [inline arpack.rb]: Yeah, I agree, but is there a reasonable way to reconcile that with the rule about not exceeding the 80th column?

Comment (sjackman) [inline arpack.rb]: I think you can break the bracket line in two with a double indentation for the second line. Also 80 is a rule of thumb, but I think it's flexible.

Comment (ilovezfs) [inline arpack.rb]: by double indentation do you mean two spaces or four

Comment (sjackman) [inline arpack.rb]: As above: 4 spaces. (I updated my comment.)

Comment (ilovezfs) [inline arpack.rb]: That makes sense in terms of trying to keep it visually separated from what's below.

But it looks like this occurs in two cases in homebrew/core, each with a three space indentation:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/emscripten.rb#L76-L77
https://github.com/Homebrew/homebrew-core/blob/master/Formula/android-sdk.rb#L39-L41

It turns out this is actually enforced by `brew audit --strict`, (i.e., by `brew style`):
```
== arpack.rb ==
C: 63: 12: Align the elements of an array literal if they span more than one line.
```

Comment (sjackman) [inline arpack.rb]: Oh, right. I'm confusing my C style with my Ruby style. 3 spaces is fine, as is the following. Use whichever you prefer.
```ruby
%w[
    pcndrv1 pdndrv1 pdndrv3 pdsdrv1
    psndrv1 psndrv3 pssdrv1 pzndrv1].each do |slv|
  system "mpirun", "-np", "4", slv
end
```

**Comment by dpo** [comment]
Version 3.4.0 is out now.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@dpo 3.4.0 looks good to go as well. Full test-bot run: https://gist.github.com/ilovezfs/4c43bccba89a327604ac11231d6682d6

I'd like to ship 3.3.0 for completeness first though.

**Comment by dpo** [comment]
Thanks. I'm all for completeness but quite a bit needs to be rebottled on arpack updates:
```
homebrew/science/armadillo        homebrew/science/getdp            homebrew/science/slepc
homebrew/science/calculix-ccx     homebrew/science/octave
homebrew/science/dealii           homebrew/science/shogun
```

**Comment by ilovezfs** [comment]
Hold off on the rebottling of those then. We'll ship this and then include revision bumps for those in the next one.

**Comment by dpo** [comment]
Sounds like a plan.

**Comment by ilovezfs** [comment]
@dpo shipped!

**Pull request closed without merging**